inst/unitTests/test_ops.R

```diff
@@ -65,8 +65,8 @@ test_compare_nanotime_ANY <- function() {
   checkTrue(nanotime(1) == as.integer64(1))
   checkTrue(nanotime(1) == 1L)
   checkTrue(nanotime(1) == 1)
-  checkTrue(is.na(nanotime(1) == "a"))  # same as 'integer64'... TODO: check
-  
+  checkException(is.na(nanotime(1) == "a"))  # now passes on and gets parse error
```

Comment:
Excellent, like for `POSIXct`:

~~~ R
> as.POSIXct("2018-12-28T16:34") == "a"
Error in as.POSIXlt.character(x, tz, ...) : 
  character string is not in a standard unambiguous format
~~~

Reply:
Yep :)